21/README.md

```diff
@@ -0,0 +1,60 @@
+```
+shortname: 21/PYTHON-ORM-DRIVER
+name: Orm implementation for official bigchaindb python driver
+type: Standard
+status: raw
+editor: Manan Patel <manan@bigchaindb.com>
+```
+
+# Abstract
+
+This is a proposal to implement ORM based wrapper for official bigchaindb [python driver](https://github.com/bigchaindb/bigchaindb-driver). Official driver provides bare bones pyhton API on top of HTTP API to connect to BigchainDB network, but ORM implementation's aim is to provide a CRAB based operational model on top of official python API in order to simplify operations on bigchainDB network - thereby improving community adoption and engagement.  
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14) \[[RFC2119](https://tools.ietf.org/html/rfc2119)\] \[[RFC8174](https://tools.ietf.org/html/rfc8174)\] when, and only when, they appear in all capitals, as shown here.
+
+# Motivation
+
+The need for this implementation is driven by following rationale -
+
+* To provide simpler API to interact and perform operations on bigchainDB. Since, traditionally ORM based CRUD (Create, Read, Update, Delete) are the most used basic operations for the database (SQL and NoSQL), CRAB model for blockchains (bigchainDB in this case) reduces the learning curve for non-blockchain developers and improves community engagement. This is specifically true for python developers as Django (one of the most popular python framework) incorporates ORM model to perform CRUD operations.
+* Our [ORM implementation](https://github.com/bigchaindb/js-driver-orm) of [javascript driver](https://github.com/bigchaindb/js-bigchaindb-driver) has recently shown good traction and usage within community.
+* We would like to keep consistency with implementations related to both - javascript and python, offical drivers.  
+
+# Specification
+
+This Orm should be implemented on top of python driver - [https://github.com/bigchaindb/bigchaindb-driver](https://github.com/bigchaindb/bigchaindb-driver).
+This implementation SHOULD include following **CRAB** operations -
+
+1. **C**reate - performs CREATE transaction with asset payload into bigchainDB.
```

Comment:
Since this is a specification for a Python ORM, it should include specific names and signatures (argument lists) for the functions, classes, methods or whatever must be implemented. The names and signatures could be inspired by the ones used in the JavaScript ORM, but of course names should be "Pythonic."

https://www.python.org/dev/peps/pep-0008/#naming-conventions